src/Hint/Typecheck.hs

```diff
@@ -37,17 +37,30 @@ kindOf type_expr =
        -- kind of errors
        failOnParseError parseType type_expr
        --
-       kind <- mayFail $ runGhc1 typeKind type_expr
+       (_, kind) <- mayFail $ runGhc1 typeKind type_expr
        --
        kindToString (Compat.Kind kind)
 
+-- | Returns a string representation of the normalized type expression.
+-- This is what @:kind!@ GHCi command prints after @=@.
```

Comment:
I realize that `after =` is confusing (but so is `:kind!`, because it does two things at once).
What about `The corresponding GHCi command is @:kind!@`?